.gitattributes

```diff
@@ -10,12 +10,8 @@
 /.gitignore        export-ignore
 /behat.yml.dist    export-ignore
 /box.json          export-ignore
-/CHANGES-v2.md     export-ignore
-/CHANGES-v3.md     export-ignore
-/CHANGES-v4.md     export-ignore
-/CHANGES-v5.md     export-ignore
-/CHANGES-v6.md     export-ignore
-/CHANGES-v7.md     export-ignore
+/CHANGES*.md       export-ignore
+/CHANGES.md        export-ignore
```

Comment:
don't need this. already covered by `/CHANGES*.md `, isn';t it?

Reply:
Let me double check...

Reply:
You were right, I fixed it.